test/DynamoCoreWpfTests/DynamoCoreWpfTests.csproj

```diff
@@ -192,6 +192,7 @@
       <Name>DynamoUtilities</Name>
       <Private>False</Private>
     </ProjectReference>
+    <ProjectReference Include="..\Libraries\PackageManagerTests\PackageManagerTests.csproj" />
```

Comment:
Added a test for loading a package and then publishing the same package with a new version.